repo/packages/E/elasticsearch/0/marathon.json

```diff
@@ -0,0 +1,18 @@
+{
+  "id": "{{elasticsearch.framework-name}}",
+  "cpus": "{{elasticsearch.scheduler.cpu}}",
+  "mem": "{{elasticsearch.scheduler.ram}}",
+  "instances": "{{elasticsearch.scheduler.instances}},
+  "args": ["-zk", "{{mesos.master}}", "-n", "{{elasticsearch.executor.instances}}", "-m", "{{elasticsearch.port}}", "-ram", "{{elasticsearch.executor.ram}}"],
+  "env": {
+    "JAVA_OPTS": "{{elasticsearch.scheduler.java-heap}}"
+  },
+  "container": {
+    "type": "DOCKER",
+    "docker": {
+      "image": "mesos/elasticsearch-scheduler",
+      "network": "HOST",
+      "forcePullImage": true
+    }
+  }
```

Comment:
Health checks added. Not 100% about how the mapping to scheduler port works, so this will have to be tested.